hbase-shaded/hbase-shaded-testing-util/pom.xml

```diff
@@ -37,43 +37,8 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>${hadoop.version}</version>
             <type>test-jar</type>
             <scope>compile</scope>
-            <exclusions>
```

Comment:
I don't have a strong opinion about this so will approve the PR but it would be better to stick with just the changes that are implied by the description _[HBASE-26691 Replacing log4j with reload4j for branch-2.x]_